servicex/resources/preflight_check.py

```diff
@@ -56,7 +53,5 @@ def post(self, request_id):
             }
 
         except Exception:
-            exc_type, exc_value, exc_traceback = sys.exc_info()
-            traceback.print_tb(exc_traceback, limit=20, file=sys.stdout)
-            print(exc_value)
+            self.logger.exception("Got exception in pre-flight check")
```

Comment:
I think the idiomatic way to log an exception with traceback is:
```
except Exception as e:
    logging.error(e, exc_info=True)
```

Reply:
Nevermind, I misunderstood how `logger.exception` works. It seems like the way you have it will produce a full traceback.